packages/discord.js/src/structures/Presence.js

```diff
@@ -63,7 +63,7 @@ class Presence extends Base {
 
   /**
    * The member of this presence
-   * @type {?GuildMember}
+   * @type {?(GuildMember | PartialGuildMember)}
```

Comment:
Here `this.member` should be turned into a property created from data passed in constructor instead of a getter. That way it could contain a partial guild member without putting that partial data into cache.